### PR TITLE
change parent toc node to overview

### PR DESCRIPTION
**Pull request by yishengjin1413**

### CODE REVIEW

File: docfx_yaml/extension.py [hunk 1]
```diff
@@ -749,7 +749,8 @@ def convert_module_to_package_if_needed(obj):
                 found_node = find_node_in_toc_tree(toc_yaml, parent_level)
 
                 if found_node:
-                    found_node.setdefault('items', []).append({'name': uid, 'uid': uid})
+                    found_node.pop('uid', 'No uid find')
```
Comment (joelmartinez) [inline docfx_yaml/extension.py]: I think this should say, "No uid found"?

**Review by joelmartinez — CHANGES_REQUESTED** [review]
Just a very small grammar change

File: docfx_yaml/extension.py (hunk 1 above)
Comment (yishengjin1413) [reply docfx_yaml/extension.py]: I have a new test build is still running, will check whether these changes work tomorrow.

**Review by joelmartinez — CHANGES_REQUESTED** [review]
Can you include a sample TOC to show how this is changing?

**Comment by yishengjin1413** [comment]
Sure. Here is the [test pages](https://review.docs.microsoft.com/en-us/python/api/adal/?branch=toc-overview1&view=azure-python). Please have a check. 

And after testing, I found that we are now blocked by JoinToc plugin, since after we change the toc structure, the [service pages](https://review.docs.microsoft.com/en-us/python/api/overview/azure/activedirectory/client?view=azure-python&branch=toc-overview1) generated by JoinToc can't show children. Tianqi is helping on fix this from JoinToc part. So I will do another test after the plugin fixed.

**Comment by yishengjin1413** [comment]
@joelmartinez This PR is ready to go now. Tianqi have fixed the bugs from Jointoc side, and we did a test with a pre-release package. You can did some check here: https://review.docs.microsoft.com/en-us/python/api/overview/azure/activedirectory/client?view=azure-python&branch=toc-overview1

**Pull request merged**